### ISSUE TITLE
Bug: create tmp file error

### ISSUE THREAD

**Issue by MVPAIR** [elvis972602/Kemono-scraper#50]
Hi,
I couldn't download this post: https://kemono.su/fantia/user/4798/post/1424682.
It always showed:  create tmp file error: open XXXXXX.jpg.tmp: The system cannot find the path specified., retry after X.0 seconds...

**Comment by Triramama** [comment]
I think the filename on that post is too long to fit in windows path, 182 symbols, script creates a subfolder for the files but it's cut short because of limitation and therefore files do not have a folder to go into

**Comment by Triramama** [comment]
nope, nothing to do with windows file path, but i still believe it has something to do with the post name, maybe the script bug that does not support large post names, smh where tf is the maintainer went off to
